tests/Makefile

```diff
@@ -1,6 +1,3 @@
-MODULE_LINT=/usr/share/moduleframework/tools/modulelint.py
```

Comment:
I don't understand what's the point of having such makefile.

Reply:
MTF provides a binary mtf which can call tests directly. https://github.com/fedora-modularity/meta-test-family/pull/101 Now we can delete Makefile.

Reply:
\o/